### PR TITLE
Feature/toolbar styles

### PR DESCRIPTION
**Pull request by felskia**

### CODE REVIEW

**Comment by felskia** [comment]
@drewbo will you look this one over. Getting a weird error. 

Thanks!

**Comment by drewbo** [comment]
@felskia can you resolve the merge conflict before I review? Thanks!

**Comment by felskia** [comment]
@drewbo ok, should be set.

File: app/scripts/components/map/index.js [hunk 1]
```diff
@@ -331,47 +331,99 @@ export const Map = React.createClass({
...
+              <li className='toggle__item'>
+                <a href="#" onClick={this.toggleVisibility.bind(this, 'all')}>
+                  <icon className='visibility'><span>Hide/Show</span></icon>
+                  <span className='line-description'>All lines {hidden.length >= 1 ? '(show all)' : '(hide all)'}</span>
+                </a>
+              </li>
+              <li className='toggle__item'>
+                <a href="#" onClick={this.toggleVisibility.bind(this, INCOMPLETE)}>
+                  <icon className='visibility'><span>Hide/Show</span></icon>
+                  <span className='line-description'>Incomplete {hidden.indexOf(INCOMPLETE) > -1 ? '(hidden)' : '(showing)'}</span>
```
Comment (drewbo) [inline app/scripts/components/map/index.js]: @felskia looks like we have two incomplete toggles and lost the completed one

File: app/styles/_map.scss [hunk 2]
```diff
@@ -1,9 +1,201 @@
...
+  }
+}
+
+.select-wrapper {
+  position: relative;
+  display: inline-block;
+  &:after {
+    position: absolute;
+    content: '';
+    background-image: url(../graphics/layout/chevron-down.svg);
```
Comment (drewbo) [inline app/styles/_map.scss]: @felskia this is still showing up even when the select button isn't there

File: app/scripts/config/base.js [hunk 3]
```diff
@@ -1,5 +1,4 @@
 module.exports = {
   environment: 'development',
-  baseUrl: 'http://localhost:4030',
-  existingRoadsSource: 'http://openroads-tiles.s3-website-us-east-1.amazonaws.com/{z}/{x}/{y}.vector.pbf'
```
Comment (drewbo) [inline app/scripts/config/base.js]: @felskia let's not drop this

File: app/styles/_map.scss (hunk 2 above)
Comment (drewbo) [inline app/styles/_map.scss]: Probably requires a JSX change since it seems like this should be on the wrapper

**Comment by drewbo** [comment]
@felskia looking great, I requested a few fixes but some might require some underlying JSX changes, let me know if you have questions there. I can also help hook up the buttons and remove the old map buttons

File: app/styles/_map.scss (hunk 2 above)
Comment (felskia) [inline app/styles/_map.scss]: Yeah, so I want it to always show up and just have the disabled look to it. Not sure how it's hidden now.

File: app/scripts/config/base.js (hunk 3 above)
Comment (felskia) [inline app/scripts/config/base.js]: I actually don't know how I did drop it :\

**Comment by felskia** [comment]
@drewbo ok, should be good. Let me know what you think.

**Comment by drewbo** [comment]
@felskia added functions to the new buttons and removed the old; also updated the select element styling. Can you review both of those? The only other thing this needs before merging is consistency between the text of line visibility toggles and the different statuses. Should we changed "In progress" to "Edited" to match?

**Pull request merged**